src/main/osd/osd_elements.c

```diff
@@ -781,11 +781,11 @@ static void osdElementCompassBar(osdElementParms_t *element)
 //display custom message from MSPv2
 static void osdElementCustomMsg(osdElementParms_t *element)
 {
-    unsigned msgIndex = element->item - OSD_CUSTOM_MSG1;
-    if (strlen(pilotConfig()->message[msgIndex]) == 0) {
+    int8_t msgIndex = element->item - OSD_CUSTOM_MSG0;
```

Comment:
```suggestion
    int msgIndex = element->item - OSD_CUSTOM_MSG0;
```

types other than 32bits (int) are slightly slower on ARM gcc